### DISCUSSION TITLE
Week 10 Project Update

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#207] [General]
I was able to make substantial progress on my menu project this week. I added all the menu items and their pricing and inserted a personal image. The bones are looking good. Now, I'd like to do more styling to the page and make it look the way I want it to. 

New Project Tasks:
- insert business logos
- insert additional images
- add business contact info
- use CSS to style page further

**Comment by Skye404** [comment]
@audreyhki how did you upload a personal image? I'm been running into challenges all the different ways I try it.

**Reply by audreyhki** [reply]
I'm sure you were able to figure it out a long time ago, but in case you didn't lol.... 

I created an images folder on my project page as well as in codespaces. In codespaces once you've created the images file you can right-click on it and select "upload" then browse your computer and upload any images you'd like to the file. Then it's only a matter of adding the image into your index.html which would look something like: 
< img src="images/ the-name-of-your-image.png" />

**Reply by Skye404** [reply]
Thanks. I was able to get it finally. I always run into issues when trying to add external sources though, like images and PDFs.

**Comment by CJ3315** [comment]
Great work on your updates!
